tsup.config.ts

```diff
@@ -1,11 +1,11 @@
 import { defineConfig } from "tsup";
 
 export default defineConfig({
-  entry: ["src/main"],
+  entry: ["src/main/index.ts"],
   outDir: "build",
   format: ["cjs", "esm"],
   target: "es2019",
   sourcemap: true,
   dts: true,
-  splitting: false,
+  clean: true,
```

Comment:
This attribute just ensures the build directory is cleaned before every build. I noticed you had that as a prebuild script, so I figured you might as well just let the build tool take care of it instead!

Reply:
I'm all for that, good change!